roles/icpc_host_backup/tasks/main.yml

```diff
@@ -0,0 +1,102 @@
+---
+
+- name: Ensure packages that are needed for backup hosts are installed
+  ansible.builtin.package:
```

Comment:
By using the apt module you can feed the whole list in 1 go, which removes the loop (and the `n` times dependency checking)

Reply:
thanks, i made that change to this one and icpc_host_packages.